apps/chat-e2e/src/core/dialSharedWithMeFixtures.ts

```diff
@@ -117,8 +115,14 @@ const dialSharedWithMeTest = dialTest.extend<{
   additionalShareUserSystemPromptListAssertion: PromptListAssertion;
   additionalShareUserEntitySettingAssertion: EntitySettingAssertion;
   additionalShareUserAttachFilesModal: AttachFilesModal;
-  additionalShareUserShareErrorToastAssertion: ShareErrorToastAssertion;
+  additionalShareUserErrorToastAssertion: ErrorToastAssertion;
 }>({
+  errorToastAssertion: async ({ additionalShareUserErrorToast }, use) => {
```

Comment:
I guess the name should be `additionalShareUserErrorToastAssertion` as declared in the line 118

Reply:
done